next.config.js

```diff
@@ -1,9 +1,18 @@
+const withPlugins = require('next-compose-plugins')
 const withSvgr = require('next-svgr')
-const withMDX = require('@next/mdx')({
-  extension: /\.mdx?$/,
-})
-module.exports = withSvgr(
-  withMDX({
-    pageExtensions: ['js', 'jsx', 'md', 'mdx', 'tsx', 'ts'],
+const withMdxEnhanced = require('next-mdx-enhanced')
+
+module.exports = withPlugins([
```

Comment:
I deleted the `pageExtensions` line from the previous config because I didn't see it in the example config I found at:

https://github.com/franky47/francoisbest.com/blob/34630038315b5bb9d944ec8b4ac2284378b3a72b/next.config.js